junebug/tests/test_workers.py

```diff
@@ -49,7 +49,7 @@ def setUp(self):
 
         app_config = {
             'transport_name': 'testtransport',
-            'mo_message_url': self.url,
+            'mo_message_url': unicode(self.url),
```

Comment:
We should probably use `.decode("utf-8")` instead of `unicode(...)` which uses the system encoding (which could be almost anything but is probably ASCII).